Calendar/CalLogHelpers/Build-CalLogCSV.ps1

```diff
@@ -0,0 +1,266 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# Build-CalLogCSV.ps1
+# This script is used to support the Get-CalendarDiagnosticObjectsSummary.ps1 script.
```

Comment:
Keep in mind that this will be included in script in the build process. So, this line will be repeated over and over again, which might not look the best. Our Copyright lines are removed so they aren't repeated.

Reply:
Done.